### PR TITLE
feat(tokens): add primitive size tokens

### PR DESCRIPTION
**Pull request by matthewferry**
## Summary

Adds a new primitive for `size`. The aim of these primitives is to use them for spacing (spacing can becomes size aliases), width/heights (e.g. avatar and icon sizes should be aliases to these primitives).

### CODE REVIEW

File: packages/tokens/src/viewport.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 viewport:
   mobile:
-    value: 0
+    value: '{ size.0.value }'
   tablet:
     value: 740px
   desktop:
     value: 992px
```
Comment (matthewferry) [inline packages/tokens/src/viewport.yaml:7]: These are a little non-standard... with the new primitive scale based on our 4px scaling, we could use 720 for tablet and 960 for desktop (or 1120). But we can also keep these as is if we'd rather not mess with the responsive sizes.

Comment (matthewferry) [reply packages/tokens/src/viewport.yaml:7]: or probably better idea to instead leave these as hard coded pixel values for now and we can look at breakpoints separately

Comment (Niznikr) [reply packages/tokens/src/viewport.yaml:7]: No strong preference either way. wdyt @apucacao since you worked on breakpoints a bit for the product?

Comment (apucacao) [reply packages/tokens/src/viewport.yaml:7]: i think leaving them as-is if it doesn't impede our progress with the new design system stuff here makes sense to me.

Comment (matthewferry) [reply packages/tokens/src/viewport.yaml:7]: Cool let's leave as defined for now :)

File: packages/tokens/src/border.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
     medium:
-      value: 0.375rem
+      value: '{ size.6.value }'
   width:
     100:
-      value: 0rem
+      value: '{ size.0.value }'
     200:
-      value: 0.063rem
+      value: '{ size.0.value }'
```
Comment (Niznikr) [inline packages/tokens/src/border.yaml:11]:
```suggestion
      value: '{ size.1.value }'
```
Or was it intentionally the same as `100`?

Comment (matthewferry) [reply packages/tokens/src/border.yaml:11]: oh jeez, nice catch!

**Review by Niznikr — APPROVED** [review]

**Pull request merged**